modules/ROOT/pages/licensing-overview.adoc

```diff
@@ -10,9 +10,19 @@ Anypoint API Community Manager licenses are procured and provisioned by MuleSoft
 . Five (5) admin users (community administrators or operators)
 . Connectivity to one (1) Anypoint Platform top-level organization
 . Up to 10,000 member accounts across all communities
+. Salesforce Standard Objects, Features, Capability, and Custom Objects in Customer Community Plus 
+. One (1) megabyte storage per user for all communities
+. 10 megabyte storage for organization and session caches
+. 1,000,000 page views per month across all communities
+. 40 gigabit network bandwidth in production for all communities and 1 gigabit network bandwidth for all sandboxes
```

Comment:
Add "environments" after production and sandbox (es) ?

Reply:
yes - agree

Reply:
Also much better! Thank you.